kube/values.go

```diff
@@ -69,9 +69,20 @@ func MakeValues(roleManifest *model.RoleManifest, defaults map[string]string) (h
 		sizing.Add(makeVarName(role.Name), entry.Sort(), helm.Comment(role.GetLongDescription()))
 	}
 
+	registry := settings.Registry
+	if registry == "" {
```

Comment:
Why do we need this again? (Comment please, either in code or as a `Comment()` for the node)